src/pages/MemberSettingsPage/ChangePassword/index.jsx

```diff
@@ -4,34 +4,32 @@ import { useFormik } from "formik";
 import * as yup from "yup";
 import useMediaQuery from "@mui/material/useMediaQuery";
 import ArrowBackIcon from "@mui/icons-material/ArrowBack";
-import { useNavigate } from "react-router-dom";
+import { useNavigate, Link } from "react-router-dom";
 import Sidebar from "../../../dashboard/SideBar";
-import { Link } from "react-router-dom";
+import firebase from "firebase/app";
 
 const ChangePasswordPage = () => {
   const navigate = useNavigate();
 
-  const handleBackClick = () => {
-    history.push("/MemberSettings");
-  };
-
   const schema = yup.object().shape({
-    currentPassword: yup.string().required("Current password is required"),
-    newPassword: yup
-      .string()
-      .required("New password is required")
-      .min(8, "Password must be at least 8 characters"),
-    confirmPassword: yup
-      .string()
-      .required("Confirm password is required")
-      .oneOf([yup.ref("newPassword"), null], "Passwords must match"),
+    email: yup.string().email("Invalid email").required("Email is required"),
   });
 
   const formik = useFormik({
     initialValues: {
-      currentPassword: "",
-      newPassword: "",
-      confirmPassword: "",
+      email: "",
+    },
+    validationSchema: schema,
+    onSubmit: (values) => {
+      firebase
+        .auth()
+        .sendPasswordResetEmail(values.email)
+        .then(() => {
+          console.log("Password changed successfully!");
+        })
+        .catch((err) => {
+          console.error("Password change failed", err.message);
+        });
```

Comment:
Make this a function in the src/auth/services.js directory there should be a pattern there you can follow.